fighthealthinsurance/common_view_logic.py

```
@@ -239,14 +239,19 @@ def create_or_update_appeal(
+                    denial = (
+                        Denial.objects.filter(denial_id=denial_id)
+                        .filter(
+                            hashed_email=Denial.get_hashed_email(email),
+                            semi_sekret=semi_sekret,
+                        )
+                        .get()
+                    )
+                except Denial.DoesNotExist:
+                    raise Exception(
+                        f"Could not find denial {denial_id} for the provided email and secret."
                     )
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)